add docs for azure log analytics unsafeSsl parameter pr#4047 (#1041)
kedacore/keda-docs: content/docs/2.10/scalers/azure-log-analytics.md

- `cloud` - Name of the cloud environment that the Azure Log Analytics workspace belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`, Optional)
- `logAnalyticsResourceURL` - Log Analytics REST API URL of the cloud environment. (Required when `cloud` is set to `Private`, e.g. `https://api.loganalytics.azure.cn/` for `AzureChinaCloud`).
- `activeDirectoryEndpoint` - Active Directory endpoint of the cloud environment. (Required when `cloud` is set to `Private`, e.g. `https://login.chinacloudapi.cn/` for `AzureChinaCloud`).

> - `unsafeSsl` - Controls whether a client verifies the server's certificate chain and host name. (Default: `false`, Optional, This value can be a bool)

Reviewer: Nit
```suggestion
- `unsafeSsl` - Determines whether or not KEDA will verify the server certificate's chain and host name. (Default: `false`, Optional, This value can be a bool)
```
